app/src/main/res/values/strings.xml

```diff
@@ -279,43 +279,38 @@
     <string name="repo_activity">Activity</string>
 
     <!-- Issue events -->
-    <string name="issue_event_closed">Issue was closed by %1$s</string>
-    <string name="issue_event_closed_with_commit">Issue was closed by %1$s with commit [commit]</string>
-    <string name="issue_event_reopened">Issue was reopened by %1$s</string>
-    <string name="issue_event_referenced">Issue was referenced by %1$s</string>
-    <string name="issue_event_referenced_with_commit">Issue was referenced by %1$s from commit [commit]</string>
-    <string name="issue_event_assigned">Issue was assigned to [b]%2$s[/b] by %1$s</string>
-    <string name="issue_event_assigned_self">Issue was self-assigned by %1$s</string>
-    <string name="issue_event_unassigned">Issue was unassigned from [b]%2$s[/b] by %1$s</string>
-    <string name="issue_event_unassigned_self">Issue was self-unassigned from %1$s</string>
-    <string name="issue_event_labeled">Label [label] was added by %1$s</string>
-    <string name="issue_event_unlabeled">Label [label] was removed by %1$s</string>
-    <string name="issue_event_milestoned">Milestone [b]%1$s[/b] was set by %2$s</string>
-    <string name="issue_event_demilestoned">Milestone [b]%1$s[/b] was removed by %2$s</string>
-    <string name="issue_event_renamed">Title was changed from [b]%1$s[/b] to [b]%2$s[/b] by %3$s</string>
-    <string name="issue_event_locked">Issue was [b]locked[/b] by %1$s</string>
-    <string name="issue_event_unlocked">Issue was unlocked by %1$s</string>
+    <string name="issue_event_closed">[b]%1$s[/b] closed this issue [time]</string>
+    <string name="issue_event_closed_with_commit">[b]%1$s[/b] closed this issue in [commit] [time]</string>
+    <string name="issue_event_reopened">[b]%1$s[/b] reopened this issue [time]</string>
+    <string name="issue_event_referenced">[b]%1$s[/b] referenced this issue [time]</string>
+    <string name="issue_event_referenced_with_commit">[b]%1$s[/b] added a commit [commit] that referenced this issue [time]</string>
```

Comment:
Shouldn't this be '... added commit [commit] that referenced...'? The 'a' inbetween sounds wrong to me.